index.html

```diff
@@ -0,0 +1,439 @@
+                                libero officia nemo expedita sunt mollitia maxime vel facilis quisquam alias, autem
+                                accusamus, eligendi eaque temporibus illo debitis tempore!
+                            </div>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="slider-image slide-2">
+                            <div>
+                                <br><br><br>
+                                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                            </div>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="slider-image slide-3">
+                            <div>
+                                <br><br><br>
+                                Veritatis odit exercitationem
+                                libero officia nemo expedita sunt mollitia maxime vel facilis quisquam alias.
+                            </div>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="services-items-container">
+            <div class="services-items-card yellow-card">
+                <a href="#">
+                    <div class="circle"><i class="fas fa-bolt"></i></div>
+                    <h3>Fast Servers</h3>
+                    <p>libero officia nemo expedita sunt mollitia maxime vel</p>
+                </a>
+            </div>
+            <div class="services-items-card blue-card">
+                <a href="#">
+                    <div class="circle"><i class="fas fa-cloud"></i></div>
+                    <h3>Cloud Servers</h3>
+                    <p>libero officia nemo expedita sunt mollitia maxime vel</p>
+                </a>
+            </div>
+            <div class="services-items-card gray-card">
+                <a href="#">
+                    <div class="circle"><i class="fas fa-arrows-alt"></i></div>
+                    <h3>Domain Transfer</h3>
+                    <p>libero officia nemo expedita sunt mollitia maxime vel</p>
+                </a>
+            </div>
+            <div class="services-items-card orange-card">
+                <a href="#">
+                    <div class="circle"><i class="fas fa-comments"></i></div>
+                    <h3>Live Support</h3>
+                    <p>libero officia nemo expedita sunt mollitia maxime vel</p>
+                </a>
+            </div>
+        </div>
+    </main>
+    <footer>
+        <nav class="menu">
+            <div class="logo"></div>
+            <div><a href="#">HOME</a></div>
+            <div><a href="#aboutUs">ABOUT US</a></div>
+            <div><a href="#">BLOG</a></div>
+            <div><a href="#">POLICY</a></div>
+            <div><a href="#contact">CONTACT</a></div>
+        </nav>
+        <main class="footer-container" id="support">
```

Comment:
CAN WE HAVE MAIN INSIDE FOOTER AND IN COMMON HOW MUCH MAIN CAN WE HAVE ON A PAGE ?